documentation/site/content/samples/azure-kubernetes-service/model-in-image.md

```diff
@@ -79,7 +101,7 @@ helm repo add weblogic-operator https://oracle.github.io/weblogic-kubernetes-ope
 ```
 $ helm install weblogic-operator weblogic-operator/weblogic-operator \
   --namespace sample-weblogic-operator-ns \
-  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:4.0.6 \
+  --set image=ghcr.io/oracle/weblogic-kubernetes-operator:4.1.7 \
```

Comment:
Similarly to the above, you can use `{{< latestVersion >}}` and this will be replaced with the most recently released version of the operator.

If you want to call out a specific version that you have tested (and where you will have to keep it up to date, as here) then preserve the specific version.

Reply:
Hello @rjeberhard  here we want to keep the tested version by specifying the version number.